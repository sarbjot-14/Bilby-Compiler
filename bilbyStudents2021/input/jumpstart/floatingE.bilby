main{

	print 1.2E2 $n;
	print 1.2E+2 $n;
	print 1.2E-2 $n;
	print 1.2E+21 $n;
}