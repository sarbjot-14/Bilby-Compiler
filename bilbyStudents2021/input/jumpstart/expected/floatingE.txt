120.000000
120.000000
0.012000
1200000000000000000000.000000
